tests/TokenizerTest.php

```diff
@@ -7,9 +7,39 @@
 use Doctrine\SqlFormatter\Tokenizer;
 use PHPUnit\Framework\Attributes\DoesNotPerformAssertions;
 use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+use function array_filter;
+use function preg_match;
+use function strtoupper;
 
 final class TokenizerTest extends TestCase
 {
+    /**
+     * @param 'reserved'|'reservedToplevel'|'reservedNewline' $propertyName
+     *
+     * @return list<string>
+     */
+    private function getTokenizerList(string $propertyName): array
+    {
+        $tokenizerReflClass = new ReflectionClass(Tokenizer::class);
+        /** @var list<string> $res */
+        $res = $tokenizerReflClass->getProperty($propertyName)->getDefaultValue();
+
+        return $res;
+    }
+
+    public function testKeywordsReservedAreUpperCasedAndWithoutWhitespace(): void
```

Comment:
I think that this test is testing something "too" internal. they way how properties are used in the class should not be IMO really tested. changing the implementation at this level of depth should not really impact tests IMO

Reply:
explained in https://github.com/doctrine/sql-formatter/pull/107#discussion_r1609798033